siibra/Atlas.m

```diff
@@ -0,0 +1,21 @@
+classdef Atlas
```

Comment:
Do you imagine this will ever be constructed directly by end user or only by the factory function `getAtlas()`?

If only constructed by factory, then I'd suggest to create a package folder for the non-user-constructed objects, e.g. `+siibra.items` or something like that. The thinking would be to keep root folder scoped to the key functions/classes users will call and read themselves.

Reply:
I agree that we should scope the non-user-constructed objects. I would like to postpone this scoping until the lib is more mature. Then it will be easier to name all non-user-constructed objects. What do you think?